packages/@react-spectrum/calendar/src/CalendarCell.tsx

```diff
@@ -49,8 +60,11 @@ export function CalendarCell({state, currentMonth, ...props}: CalendarCellProps)
   let isSelectionEnd = isSelected && highlightedRange && isSameDay(props.date, highlightedRange.end);
   let {locale} = useLocale();
   let dayOfWeek = getDayOfWeek(props.date, locale);
```

Comment:
Could we update `getDayOfWeek` similarly to how we updated `startOfWeek` etc. to take a parameter for the first day of the week? Then we wouldn't need to export `getWeekStart` and this logic could be reused in RAC, hooks, etc.

Reply:
Actually looks like this logic is missing in RAC right now. If we moved it into internationalized we'd get it for free right?